apps/src/p5lab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -162,6 +168,57 @@ class AnimationPicker extends React.Component {
     );
   }
 
+  /**
+   * Send the uploaded image file to be moderated. Then continue with uploadStart.
+   */
+  handleModeratedUploadStart = data => {
+    const file = data.files[0];
```

Comment:
could this throw a null pointer?

Reply:
Ah yes, it could throw:
`Uncaught TypeError: Cannot read properties of null (reading 'files')`
I'll add a guard here!
```
const file = data?.files?.[0];
if (!file) {
  console.error('No file found in upload data.');
  return;
}
```
Just noting that `data.files[0]` is assigned in the existing `onUploadStart`, but not adding since it's only called after `handleModeratedUploadStart`.